sky/clouds/service_catalog/common.py

```diff
@@ -46,15 +46,14 @@ def get_catalog_path(filename: str) -> str:
     return os.path.join(_CATALOG_DIR, filename)
 
 
-def read_catalog(filename: str) -> pd.DataFrame:
+def read_catalog(filename: str, cloud: cloud_lib.Cloud) -> pd.DataFrame:
```

Comment:
nit: `cloud` is used for printing a string name; can we pass `cloud_display_name: str`? Or use the dir name of `filename`.